alacritty/src/display/mod.rs

```diff
@@ -1620,7 +1749,8 @@ fn window_size(
     let grid_width = cell_width * dimensions.columns.0.max(MIN_COLUMNS) as f32;
     let grid_height = cell_height * dimensions.lines.max(MIN_SCREEN_LINES) as f32;
 
-    let width = (padding.0).mul_add(2., grid_width).floor();
+    let width = (padding.0).mul_add(2., grid_width).floor()
+        + config.scrollbar.additional_padding(scale_factor);
```

Comment:
I'm not sure this takes the existing UI lines into account?

Reply:
Could you clarify what you mean? I think the window_size is always independent of what UI lines we currently have. The scrollbar padding is just added in the same way as other padding, and only depends config and window scale_factor.